models/topics.model.js

```diff
@@ -12,3 +12,15 @@ exports.fetchEndpoints = async () => {
   const parsedResult = JSON.parse(result);
   return parsedResult;
 };
+
+exports.fetchArticleById = async (article_id) => {
+  let queryStr = `SELECT * FROM articles WHERE article_id = $1`;
+  let queryVal = [article_id];
```

Comment:
Arrays should be declared with const. Are you anticipating this query needing a query array that is accessible outside of the db.query?